### PR TITLE
upgrade to 1.16 version

### PR DESCRIPTION
**Pull request by kaixuanliu**
Upgrade tei-gaudi to match habana 1.16 driver

### CODE REVIEW

**Comment by kaixuanliu** [comment]
@regisss , pls help review

File: Dockerfile-hpu [hunk 1]
```diff
@@ -54,7 +54,7 @@ COPY proto proto
 
 RUN cargo build --release --bin text-embeddings-router -F grpc -F python --no-default-features && sccache -s
 
-FROM vault.habana.ai/gaudi-docker/1.15.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest as base
+FROM vault.habana.ai/gaudi-docker/1.16.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest as base
```
Comment (igordcard) [inline Dockerfile-hpu:58]: Looks like https://vault.habana.ai/ui/native/gaudi-docker/1.16.2 is out.

Or maybe officially only 1.16.1: https://docs.habana.ai/en/v1.16.1/

**Review by igordcard — APPROVED** [review]

File: Dockerfile-hpu (hunk 1 above)
Comment (kaixuanliu) [reply Dockerfile-hpu:58]: Have change it to 1.16.1.

**Comment by yao-matrix** [comment]
@regisss, could you help review and merge? We are about to claim rls to support 1.16. Thx.

**Review by regisss — APPROVED** [review]

**Pull request merged**